spago.dhall

```diff
@@ -1,5 +1,6 @@
 { name = "js-uri"
-, dependencies = [ "console", "effect", "psci-support" ]
+, dependencies =
+  [ "console", "effect", "functions", "maybe", "psci-support", "strings" ]
```

Comment:
I can drop the `strings` dependency if I apply replacements in JavaScript instead of in PureScript (for the `*FormURLComponent` functions).

Reply:
I'm not against this. This library already has some non-trivial FFI bindings (well, they're only just non-trivial :smile:) so doing it all in the FFI is fine to me.